helm-charts/whitesource-renovate/templates/pvc.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if and .Values.cachePersistence.enabled (not .Values.cachePersistence.existingClaim) }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ include "whitesource-renovate.fullname" . }}
```

Comment:
Would there be value to adding a `-cache` suffix to this in case renovate ever needs other PVC's?
```
name: {{ include "whitesource-renovate.fullname" . }}-cache
```

Reply:
Sounds like a good idea, thanks! :)